Library/Formula/consul.rb

```diff
@@ -174,5 +184,7 @@ def install
     end
     sleep 3
     system "#{bin}/consul", "leave"
+
+    assert_equal true, File.exist?("#{pkgshare}/index.html") if build.with? "web-ui"
```

Comment:
You can just do `assert File.exist` here. I think it's safe to cut the line entirely though; we never test non-default options.